### ISSUE TITLE
Previews: some fields return published data instead of revision data

### ISSUE THREAD

**Issue by peteluffman** [wp-graphql/wp-graphql-acf#159]
I'm coming across an odd issue where certain fields aren't returning the revision data when using `asPreview: true` in a page query. Instead i'm getting the published data.

The fields in question are within a group, and consist of the following field types:

- text
- text area
- repeater

Only the textarea actually returns the correct value, which think this is due to me having another custom field using the same name outside of a group.

Digging around in the code, I found if I change the if statement on line 57 within class-config.php to use a false value or remove the statement altogether I get the correct revision data.

```
add_filter( 'graphql_resolve_revision_meta_from_parent', function( $should, $object_id, $meta_key, $single ) {
    //if ( in_array( $meta_key, $this->registered_field_names, true ) ) {
    if ( ! in_array( $meta_key, $this->registered_field_names, true ) ) {
        return false;
    }

    return $should;
}, 10, 4 );

```

Any ideas what's going on?

**Comment by CalebBarnes** [comment]
Maybe related to https://github.com/wp-graphql/wp-graphql/issues/1441 ?

**Comment by jasonbahl** [comment]
I believe this was resolved by #205. If not, please open a new issue and reference this one. 🙏
